components/ChoreFrequency.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from 'react';
+import { ScrollView, StyleSheet, TouchableOpacity, View } from 'react-native';
+
+import { Badge, Card, Text } from 'react-native-paper';
+
+const ChoreFrequency = () => {
+  const [value, setValue] = React.useState(7);
+  const [isPressed, setIsPressed] = React.useState(false);
```

Comment:
Same as above, .React isn't necessary
```suggestion
  const [value, setValue] = useState(7);
  const [isPressed, setIsPressed] = useState(false);
```

Reply:
fixed